src/test/cli/rbd/help.t

```diff
@@ -63,7 +63,8 @@
                                         associated.
       image-meta set                    Image metadata set key with value.
       import                            Import image from file.
-      import-diff                       Import an incremental diff.
+      import-diff                       Apply an incremental diff to image HEAD, 
```

Comment:
There is a stray space at the end here which is causing the test to fail:
```suggestion
      import-diff                       Apply an incremental diff to image HEAD,
```

Reply:
Also need to adjust help for `rbd import-diff` command itself, line 1258 in the same file.

Reply:
> Also need to adjust help for `rbd import-diff` command itself, line 1258 in the same file.

Unresolving -- this is still outstanding (i.e. the test would fail).

Reply:
Hi, @idryomov line 1258 has fixed. And I have no idea why `make check — make check failed`. Does it have relationship by my change?

Reply:
> Does it have relationship by my change?

It doesn't look like it.  I will retrigger it.